etc/config.xml

```diff
@@ -103,6 +103,7 @@
                 <can_void>1</can_void>
                 <can_cancel>1</can_cancel>
                 <group>adyen</group>
+                <guest_checkout_tokenisation>0</guest_checkout_tokenisation>
```

Comment:
What is the purpose of having this here?

Reply:
I removed it